1209/DC21/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Dual Charger
+owner: FPGA-Computer
```

Comment:
The capitalisation of the owner line needs to match the directory name of your organisation ('fpga-computer'). Other than that, this all looks fine.